src/ResourceManager/StorageSync/Commands.StorageSync/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.PowerShell.4.ReferenceAssemblies" version="1.0.0" targetFramework="net452" />
```

Comment:
@afedyashov what is this package?

Reply:
PowerShell SDK assemblies, https://www.nuget.org/packages/Microsoft.PowerShell.4.ReferenceAssemblies/

Reply:
Right, but what is it being used for? This isn't a package that any existing project in our repository uses.